code/API_definitions/number_verification.yaml

```diff
@@ -0,0 +1,375 @@
+openapi: 3.0.3
+info:
+  title: Number Verification
+  description: |-
+    Service Enabling Network Function API to verify that the provided **phone number** is the one used in the device. It verifies that the user is using a device with the same *phone number* as it is declared.
+    It also make it possible for a Service provider to verify the number itself by returning the phone number retrieved from the authenticated user's access token.
```

Comment:
are we assuming here that MSISDN is somehow returned in Access Token? should it be JWT? where MSISDN is then returned?

Reply:
The API doesn't return the Token. It's a three legged authentication and the phone number is associated to the token so it can be retrieved by the API in order to compare it with the input data.

Reply:
Hi @monamok ! 

this particular sentence makes confision:

>...verify the number itself by returning the phone number retrieved from the authenticated user's access token.

I read it like:

>as a sw developer i can retrieve phone number from authenticated user's access token, so that I can verify phone number.

maybe sth like this is better:

>as a sw developer I can retrieve information about phone number of the device, so that I can confirm that end user possess the device.

what do yo think?

Reply:
I rephrased it. Please take a look :)